### PR TITLE
Run regression tests on pg15 windows

### PR DESCRIPTION
**Pull request by svenklemm**
Disable-check: commit-count

### CODE REVIEW

File: src/bgw/job_stat.c [hunk 1]
```diff
@@ -272,7 +272,7 @@ static float8
 calculate_jitter_percent()
 {
 	/* returns a number in the range [-0.125, 0.125] */
-	uint8 percent = random();
+	uint8 percent = rand();
```
Comment (konskov) [inline src/bgw/job_stat.c:275]: it's not obvious to me why this change is required?

Comment (svenklemm) [reply src/bgw/job_stat.c:275]: The explanation is in the commit message of the 2nd commit:
```
Use rand() instead of random() cause the latter is not available
on Windows and postgres stopped backporting it with PG15.
Ideally we switch to the crypto functions added in PG15 but that
requires a bit more work and this is the minimal change required
to get it to build against PG15 on Windows.
```

Comment (konskov) [reply src/bgw/job_stat.c:275]: I see - sorry I made you repeat yourself here!

**Review by konskov — APPROVED** [review]

**Review by sb230132 — APPROVED** [review]
